mass_mailing_custom_unsubscribe/models/mail_unsubscription.py

```diff
@@ -46,6 +66,15 @@ def _selection_unsubscriber_id(self):
         """Models that can be linked to a ``mail.mass_mailing``."""
         return self.env["mail.mass_mailing"]._get_mailing_model()
 
+    @api.multi
+    @api.constrains("action", "reason_id")
+    def _check_reason_needed(self):
+        """Ensure reason is given for unsubscriptions."""
+        for one in self:
+            if one.action == "unsubscription" and not one.reason_id:
+                raise exceptions.ReasonRequiredError(
+                    _("Please indicate why are you unsubscribing."))
```

Comment:
The tour fails because of an [error 500](https://travis-ci.org/OCA/social/jobs/383253541#L1719). [This seems to be the cause](https://travis-ci.org/OCA/social/jobs/383253541#L1568).

Most likely, you need to alter the tour, so it indicates the unsubscription reason.

Reply:
Grrr, I can't reproduce in my local even with the whole repo CI :confused: